lua_scripts/kalimdor/tanaris/Quest_2882_Cuergo_Gold.lua

```diff
@@ -0,0 +1,56 @@
+--[[
+    EmuDevs <http://emudevs.com/forum.php>
+    Eluna Lua Engine <https://github.com/ElunaLuaEngine/Eluna>
+    Eluna Scripts <https://github.com/ElunaLuaEngine/Scripts>
+    Eluna Wiki <http://wiki.emudevs.com/doku.php?id=eluna>
+
+    -= Script Information =-
+    * Zone: Tanaris
+    * Script Type: Quest (Cuergo's Gold)
+    * Creature: Sailor boys (7899, 7901, 7902)
+    * Game object: Pirate's treasure chest (142194)
+	* Version: tested on Zero
+--]]
+
+-- This creates the table used to localize all variables and functions
+local CuergoGold = {
+    GO_INCONSPICUOUS_LANDMARK = 142189;
+    NPC_TREASURE_HUNTING_PIRATE = 7899,
+    NPC_TREASURE_HUNTING_SWASHBUCKLER = 7901,
+    NPC_TREASURE_HUNTING_BUCCANEER = 7902,
+    GO_PIRATE_TREASURE = 142194,    
+    SPAWN_DURATION = 600000 -- pirates and chest will exist in world for 10 minutes
+}
+
+function CuergoGold.PirateTreasure_OnUse(event, player, go)
+    -- despawn the chest
+    go:Despawn(10)
+end
+
+function SpawnTreasureHuntingPirates(player, totalPirates)
+    local X = player:GetX();
+    local Y = player:GetY();
+    local Z = player:GetZ();
+    for i = 1, totalPirates, 1 do
+        math.randomseed(os.time())
+        pirate = math.random(1,3)
```

Comment:
You're right, that line does not do as I wanted. I had actually deleted it from my Linux version of the script!